test/build.py

```diff
@@ -73,8 +77,8 @@ def convert_wast_to_js(out_js_dir):
 
     pool = mp.Pool(processes=8)
     for result in pool.imap_unordered(convert_one_wast_file, inputs):
-        if result != 0:
-            print('Error when compiling {} to JS: {}', wast_file, result.stdout)
+        if result.returncode != 0:
+            print('Error when compiling to JS:\n{}\n{}'.format(result.args, result.stdout))
```

Comment:
Would it make sense to print `result.stderr` if it's non empty too?

Reply:
Yeah, that would be good. I'll update.

Reply:
I've updated `run` to merge stderr and stdout, which should fix this.